docs/source/_templates/partials/submitting_to_validator.rst

```diff
@@ -49,7 +49,7 @@ prepared the BatchList:
 
 
 And here is what it would look like if you saved the binary to a file, and then
-sent it from the command-line with *curl*:
+sent it from the command line with ``curl``:
```

Comment:
Good idea!  I'll update the entire doc in my next pass. For now, this instance is consistent with several others in the doc (i.e., it's not the only one) and is grammatically correct ("command-line" for an adjective followed by a noun, vs "command line" when used as an adjective+noun). But I agree that it can - and will - be improved.